app/file_test.go

```diff
@@ -87,13 +87,34 @@ func TestDoUploadFile(t *testing.T) {
 }
 
 func TestUploadFile(t *testing.T) {
-	th := Setup(t)
+	th := Setup(t).InitBasic()
 	defer th.TearDown()
 
-	channelId := model.NewId()
+	session, _ := th.App.CreateSession(
```

Comment:
We should require that the error here is `nil`; otherwise, the test could fail and we would not know why.